observation-service/src/main/java/gov/cdc/etldatapipeline/observation/repository/model/reporting/ObservationCodedKey.java

```diff
@@ -0,0 +1,14 @@
+package gov.cdc.etldatapipeline.observation.repository.model.reporting;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
```

Comment:
Suggestion: We can add an @AllArgsConstructor and simplify the object creation with one constructor

Reply:
Then it will require to create a new instance of the key on each iteration. Not very efficient.